src/utils.h

```diff
@@ -8,7 +8,7 @@ namespace pktvisor {
 
 // list of subnets we count as "host" to determine direction of packets
 typedef std::pair<pcpp::IPv4Address, pcpp::IPv4Address> IPv4subnet;
-typedef std::pair<pcpp::IPv6Address, pcpp::IPv6Address> IPv6subnet;
+typedef std::pair<pcpp::IPv6Address, uint8_t> IPv6subnet;
```

Comment:
Using actual struct instead of `std::pair` might make the code more readable.

```cpp
struct IPv6subnet {
    pcpp::IPvAddress address;
    uint8_t mask;
}
```